requirements/common.txt

```diff
@@ -52,3 +52,4 @@ python-geoip-geolite2
 pycountry
 pytz
 diff-match-patch
+google-compute-engine
```

Comment:
Why is this needed ? We don't want to have any code dependency on google compute engine